concent_api/core/migrations/0009_auto_20180829_1224.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-08-29 12:24
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0008_store_message_timestamp_fix'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='storedmessage',
+            name='subtask_id',
+            field=models.CharField(blank=True, max_length=36, null=True),
+        ),
+        migrations.AlterField(
+            model_name='storedmessage',
+            name='task_id',
+            field=models.CharField(blank=True, max_length=36, null=True),
```

Comment:
same here

Reply:
Actually it can't be nullable nor blank. Changed